example-app/SantokuApp/jest/setup/react-native-reanimated.js

```diff
@@ -4,4 +4,4 @@ global.ReanimatedDataMock = {
   now: () => Date.now(),
 };
 // https://docs.swmansion.com/react-native-reanimated/docs/guide/testing/
-require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests();
+require('react-native-reanimated/lib/module/reanimated2/jestUtils').setUpTests();
```

Comment:
> `require('react-native-reanimated').setUpTests();`

これでもいけました！

- `node_modules/react-native-reanimated/package.json`
  - `"main": "lib/module/index",`
- `node_modules/react-native-reanimated/lib/module/index.js`
  - `export * from './reanimated2';`
- `node_modules/react-native-reanimated/lib/module/reanimated2/index.js`
  - `export * from './jestUtils';`
  - これが [v2 だとなかった](https://github.com/software-mansion/react-native-reanimated/blob/2.14.4/src/reanimated2/index.ts)

----

```diff
-https://docs.swmansion.com/react-native-reanimated/docs/guide/testing/
+https://docs.swmansion.com/react-native-reanimated/docs/guides/testing
```
全く気にしてなかったです。変更しておきます。

Reply:
> setupFilesAfterEnvでやらないとだめ？

特にエラーは出てなかったです 🤔 

`setupFiles` には入ってますね。。

```js
  setupFiles: [
    '<rootDir>/jest/setup/global.js',
    '<rootDir>/jest/setup/netinfo.js',
    '<rootDir>/jest/setup/react-native.js',
    '<rootDir>/jest/setup/react-native-gesture-handler.js',
    '<rootDir>/jest/setup/react-native-reanimated.js',
    '<rootDir>/jest/setup/react-native-safe-area-context.js',
    '<rootDir>/jest/setup/react-query.js',
    '<rootDir>/jest/setup/useFocusEffect.js',
  ],
```

[Testing with Jest | React Native Reanimated](https://docs.swmansion.com/react-native-reanimated/docs/guides/testing/#setup)

> If you use Jest in a version older than 28, you should set setupFiles property instead of setupFilesAfterEnv

現在jestは29
```
$ npm ls jest
├─┬ eslint-plugin-jest@27.2.2
│ └── jest@29.5.0 deduped
├─┬ expo-module-scripts@3.0.9
│ ├─┬ jest-watch-typeahead@2.2.1
│ │ └── jest@29.5.0 deduped
│ └─┬ ts-jest@29.0.5
│   └── jest@29.5.0 deduped
└── jest@29.5.0
```

v2でも同じでExpoと関係無かったので対応する場合でも別PRにします。

https://docs.swmansion.com/react-native-reanimated/docs/2.x/guide/testing/

----

両者の違い（調査途中）

[Jestの設定 · Jest](https://jestjs.io/ja/docs/configuration#setupfiles-array)

- setupFiles
  - 各 setupFile はテスト ファイルごとに 1 回実行されます。
  - すべてのテストは独自の環境で実行されるため、これらのスクリプトは、setupFilesAfterEnv を実行する前、およびテスト コード自体の前にテスト環境で実行されます。
- setupFilesAfterEnv
  - スイート内の各テスト ファイルが実行される前に、テスト フレームワークを構成またはセットアップするコードを実行するモジュールへのパスのリスト。
  - setupFiles はテスト フレームワークが環境にインストールされる前に実行されるため、このスクリプト ファイルは、テスト フレームワークが環境にインストールされた直後、テスト コード自体の前にコードを実行する機会を提供します。

Reply:
修正し忘れてました。

9a1dd46d4889d6f55c358414e73e12550911a491 `test: refactor: react-native-reanimatedのtest用setup関数のimport pathを短縮`